examples/rl_grid_mpc_enum.py

```diff
@@ -29,10 +30,12 @@
 
 i_ref_dq = Sequence(np.array([0, 1]), np.array([[1, 0], [1, 0]]), base.w)
 
-ctr = control.mpc.CurrCtrMpcEnum(lambda_u=10e-3,
-                                 Np=1,
-                                 Ts=100e-6,
-                                 i_ref_seq_dq=i_ref_dq)
+solver = mpc.solvers.MpcEnum(conv=conv)
```

Comment:
For line 27: When we deal with the grid, instead of setting the grid inductance and resistance, would it be better to set the short-circuit ratio and impedance ratio? Then, by setting the converter power, the inductance and resistance can be computed.

Reply:
I will make a note and we can think about this